app/scripts/proactive/templates/file-browser-template.html

```diff
@@ -28,6 +28,7 @@ <h5>
                 <a id="delete-file-btn" href="javascript:void(0)" class="pointer pa-tool-btn" title="Delete the selected file or folder">
                     <i class="fas fa-trash"></i>
                 </a>
+                <label title="Filter files according to a glob pattern" style="padding-left: 160px">Filter: <input type="text" value="*" id="filter-files"/> </label>
```

Comment:
glob or global?

Reply:
it's called a glob pattern.
https://en.wikipedia.org/wiki/Glob_(programming)

Reply:
I learned something new. Thanks!